<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Process_prog</class>
 <widget class="QWidget" name="Process_prog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Process_prog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/spiral_calendar_96px.png</normaloff>:/spiral_calendar_96px.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Programme executabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Programme_lineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Arguments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="encour_Button">
       <property name="text">
        <string>en cours</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Sortie_Button">
       <property name="text">
        <string>sortie</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAjouterArguments">
   <property name="text">
    <string>ajouter</string>
   </property>
  </action>
  <action name="actionSupprimer_argument">
   <property name="text">
    <string>supprimer argument</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
